dagshub/upload/wrapper.py

```diff
@@ -153,7 +155,7 @@ def _log_upload_details(self, data, res, files):
             logger.debug(f"Response ({res.status_code})\n")
 
         if res.status_code == 200:
-            logger.info("Upload finished successfully!")
+            print("Upload finished successfully!")
```

Comment:
Any reason for this?
It's rude of libraries to be directly `print`ing - you might screw up a user's intended output (think of programs that pipe their outputs to other programs, it has to be very precise